apps/user/services/authentication.go

```diff
@@ -181,6 +230,13 @@ func getNotFoundErrorResponse() (authenticatedUser *userSerializers.UserAuthSeri
 	return nil, http.StatusNotFound, errors.New("user not found")
 }
 
+// getInvalidEmailPasswordErrorResponse ...
+func getInvalidEmailPasswordErrorResponse() (authenticatedUser *userSerializers.UserAuthSerializer,
```

Comment:
If it is always nil why are we using `authenticatedUser` in the response.
```suggestion
func getInvalidEmailPasswordErrorResponse() (status int, err error) {
```

Reply:
so we can directly use it like `return getInvalidEmailPasswordErrorResponse()` instead of `return nil,getInvalidEmailPasswordErrorResponse()`